src/Widgets/SpaceWidget/SpaceWidgetEditingConfig.ts

```diff
@@ -1,10 +1,10 @@
 import { provideEditingConfig } from 'scrivito'
 import { SpaceWidget } from './SpaceWidgetClass'
-import { classNameToThumbnail } from '../../utils/classNameToThumbnail'
+import Thumbnail from './thumbnail.svg'
 
 provideEditingConfig(SpaceWidget, {
```

Comment:
There is a fairly new widget, which icon is still missing: SearchResultsWidget`. Please add it.